infra/recipes/terraform/azure/sandbox/outputs.tf

```diff
@@ -0,0 +1,42 @@
+output "resource_group_name" {
+  value = azurerm_resource_group.sandbox.name
+}
+
+output "kubernetes_cluster_name" {
+  value = azurerm_kubernetes_cluster.sandbox.name
+}
+
+output "client_certificate" {
+  value     = azurerm_kubernetes_cluster.sandbox.kube_config[0].client_certificate
+  sensitive = true
+}
+
+output "client_key" {
+  value     = azurerm_kubernetes_cluster.sandbox.kube_config[0].client_key
+  sensitive = true
+}
+
+output "cluster_ca_certificate" {
+  value     = azurerm_kubernetes_cluster.sandbox.kube_config[0].cluster_ca_certificate
+  sensitive = true
+}
+
+output "cluster_password" {
+  value     = azurerm_kubernetes_cluster.sandbox.kube_config[0].password
+  sensitive = true
+}
```

Comment:
Why do we add key, password, etc. in outputs.tf? 

Including sensitive information such as key certificates and passwords in the outputs.tf file is generally not a good idea due to security concerns. This information is exposed in logs, persisted in state file.

Reply:
Okay good point. I originally included for my understanding purposes, but I will remove

Reply:
Actually, because `sensitive` is set to true, is it okay to include them? https://developer.hashicorp.com/terraform/language/values/outputs#sensitive-suppressing-values-in-cli-output